rmw_zenoh_cpp/src/rmw_event.cpp

```diff
@@ -243,7 +246,10 @@ rmw_take_event(
         ei->total_count = st.total_count;
         ei->total_count_change = st.total_count_change;
         ei->current_count = st.current_count;
-        ei->current_count_change = st.current_count_change;
+        ei->current_count_change = static_cast<int32_t>(std::clamp<int64_t>(
+          st.current_count_change,
+          std::numeric_limits<int32_t>::min(),
+          std::numeric_limits<int32_t>::max()));
```

Comment:
Apply clamps in update_event_status

Reply:
clamping the value in the method call
https://github.com/ros2/rmw_zenoh/pull/500/commits/55e487ff78ae1a752677040eebac940ddafcb9d4